includes/wc-core-functions.php

```diff
@@ -1800,7 +1800,7 @@ function wc_get_rounding_precision() {
  */
 function wc_add_number_precision( $value, $round = true ) {
 	$cent_precision = pow( 10, wc_get_price_decimals() );
-	$value          = $value * $cent_precision;
+	$value          = $value == "" ? 0 : $value * $cent_precision;
```

Comment:
@RiLaRos Thanks for submitting the PR. Could you change the `0` to `0.0` so it is a floating point value instead of an integer. The integer value may trigger warnings on some sites running PHP 7.4.

Reply:
@rrennick I already made the requested changes.

Reply:
@RiLaRos Thanks. I've put this on the review list.